README.md

```diff
@@ -27,62 +27,119 @@ Canopy is parameterized by foliage distribution shape functions and parameters f
-
-
-    **Table 1. Canopy-App Required Input Variables**
-
-    | Variable Name    | Variable Description and Units                    |
-    | ---------------  | ------------------------------------------------- |  
-    | lat              | Latitude  (degrees)                               |
-    | lon              | Longitude (degrees; from 0-360)                   |
-    | time             | Timestamp (days since YYYY-N-D 0:0:0) (NetCDF Only) |
-    | fh               | Forest canopy height (m)                          |
-    | href             | Reference height above canopy (m) - 10 m          |
-    | ugrd10m          | U wind at HREF (m/s), e.g., 10 m                  |
-    | vgrd10m          | V wind at HREF (m/s), e.g., 10 m                  |
-    | clu              | Forest clumping index (dimensionless)             |
-    | lai              | Leaf area index (m2/m2)                           |
-    | vtype            | Vegetation type (dimensionless), VIIRS Only       |
-    | ffrac            | Forest fraction (dimensionless)                   |
-    | fricv            | Friction velocity (m/s)                           |
-    | csz              | Cosine of the solar zenith angle (dimensionless)  |
-    | sfcr             | Total surface roughness length (m)                |
-    | mol              | Monin-Obukhov Length (m)                          |
-    | frp              | Total Fire Radiative Power (MW/grid cell area)    |
-
-    **Table 2. Current User Namelist Options**
+    The Canopy-App input data in Table 2 below is based around NOAA's UFS operational Global Forecast System Version 16 (GFSv16) gridded met data, and is supplemented with external canopy data (from numerous sources) and other external and calculated input variables.  
+
+    **Table 2. Canopy-App Required Input Variables**
+
+    | **GFS /Met/Land/Soil Variables**    | **Variable Description and Units**                           |  **Example Data Sources/References (if necessary)**                                                |
+    | ----------------------------------  | ------------------------------------------------------------ |  ------------------------------------------------- 						      |
+    | lat                                 | Latitude  (degrees)                                          |  N/A                                               						      |
+    | lon                                 | Longitude (degrees; from 0-360)                              |  N/A                                               					              |
+    | time                                | Timestamp (days since YYYY-N-D 0:0:0) (NetCDF Only)          |  N/A                                               					              |  
+    | ugrd10m                             | U wind at HREF (m/s), e.g., 10 m                             |  UFS NOAA/GFSv16 *(see below for downloading using AWS)                                 						      |
```

Comment:
| ugrd10m                             | U wind at reference height above canopy (m/s), e.g., 10 m                             |  UFS NOAA/GFSv16 *(see below for downloading using AWS)                                 						      |